src/pages/guidelines/accessibility/overview.mdx

```diff
@@ -73,13 +98,13 @@ Low vision can include partial sight in one or both eyes and can range from mild
 #### What designers should think about
 
 - Maximize the readability and visual clarity of content.
-- Consider how relative proximity of information changes when a page is magnified.
+- Consider how the relative proximity of information changes when a page is magnified.
 - Follow [keyboard guidelines](https://www.nngroup.com/articles/keyboard-accessibility/) and test with a screen reader to ensure the page is read to the user in a logical order.
-- To get a better understanding of the various low-vision disabilities, we recommend using the [NoCoffee](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) Chrome plugin to preview websites.
+- To gain a better understanding of the various low-vision disabilities, use the [NoCoffee](https://chrome.google.com/webstore/detail/nocoffee/jjeeggmbnhckmgdhmgdckeigabjfbddl) Chrome plugin to preview websites.
 
```

Comment:
- designing for users with [cognitive and learning disabilities](https://www.w3.org/WAI/cognitive/)

Reply:
What is this in response to?

Reply:
Sorry -- super not clear in hindsight. It's an additional overarching concern for designers working accessibly.